License2Deploy/rolling_deploy.py

```diff
@@ -358,6 +393,13 @@ def enable_project_cloudwatch_alarms(self):
         logging.error("Unable to enable the cloud-watch alarm, please investigate: {0}".format(e))
         exit(self.exit_error_code)
 
+  def get_target_group(self, asg_group):
+    target_groups = self.asg.describe_auto_scaling_groups(AutoScalingGroupNames=[asg_group])['AutoScalingGroups'][0]['TargetGroupARNs']
```

Comment:
is there always one autoscaling group and one target group ?

Reply:
could be more than one, but we're picking 1st one always